invenio_communities/templates/semantic-ui/invenio_communities/details/header.html

```diff
@@ -104,14 +104,15 @@ <h1 class="ui medium header mb-0">{{ community.metadata.title }}</h1>
           </div>
         </div>
       </div>
-
       <div
         class="sixteen wide mobile sixteen wide tablet three wide computer right aligned middle aligned column">
-        <a href="/uploads/new?community={{ community.slug }}"
-           class="ui positive button labeled icon rel-mt-1 theme-secondary">
-          <i class="upload icon" aria-hidden="true"></i>
-          {{ _("New upload") }}
-        </a>
+        {%- if not community_use_jinja_header %}
```

Comment:
I might miss some context but why do we show the "New Upload" button only when we don't have a branded community (`community_use_jinja_header` flag)?

Reply:
we dont show the new upload button when we are already in the deposit form (discussed also in person)